build_components.sh

```diff
@@ -32,8 +32,7 @@ fi
 
 : ${BUILD_OPTIONS:=""}
 
-scons ${BUILD_OPTIONS} $option --config=force --update-prereq=all \
-  --build-deps=yes $*
+scons ${BUILD_OPTIONS} $option --config=force --build-deps=yes $*
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Use "$@" (with quotes) to prevent whitespace problems. [SC2048]
(lint) Double quote to prevent globbing and word splitting. [SC2086]